src/test/java/com/cx/restclient/general/AstScaTests.java

```diff
@@ -149,6 +151,14 @@ private void localDirScan(CxScanConfig config) throws MalformedURLException {
 
     private void scanRemoteRepo(String repoUrlProp, boolean useOnPremAuthentication) throws MalformedURLException {
         CxScanConfig config = initScaConfig(repoUrlProp, useOnPremAuthentication);
+
+        RemoteRepositoryInfo repoInfo = new RemoteRepositoryInfo();
+        URL repoUrl = new URL(prop("astSast.remoteRepoUrl.public"));
```

Comment:
scanRemoteRepo is called from both scan_remotePublicRepo and scan_remotePrivateRepo. However, you are using "astSast.remoteRepoUrl.public" and "astSca.remoteRepo.private.token" properties here, which is confusing.